notebooks/post-startup.sh

```diff
@@ -8,9 +8,14 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# Send stdout and stderr from this script to a tmp file for debugging.
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
```

Comment:
nit: Why is that important? A brief explanation here would be helpful.

Reply:
thanks, added more, PTAL